frontend/ui/widget/opdsbrowser.lua

```diff
@@ -474,6 +474,8 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     -- download to user selected directory or last opened dir
     local lastdir = G_reader_settings:readSetting("lastdir")
     local download_dir = G_reader_settings:readSetting("download_dir") or lastdir
+    item.author = item.author:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','')
+    item.title = item.title:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','')
```

Comment:
I think characters that look similar may be confusing. User might think that e.g question mark or colon are correct characters in file name.
We can remove incorrect characters or like @houqp replace those with `_`.
I cannot find easy way to detect filesystems in LUA so I propose to remove all those characters : `\,/,:,*,?,",<,>,|`